src/plugins/management/public/components/management_app/management_app.tsx

```diff
@@ -53,8 +57,11 @@ export const ManagementApp = ({ dependencies, history, theme$ }: ManagementAppPr
         ...(item.href ? reactRouterNavigate(scopedHistory, item.href) : {}),
       });
 
+      const managementBreadcrumb = crumbs.length
```

Comment:
Could you please add a code comment here to explain how this condition results in the intended UX? For example:

```js
// Clicking the Management breadcrumb to navigate back to the "root" only
// makes sense if there's a management app open. So when one isn't open
// this breadcrumb shouldn't be a clickable link. 
```

I'm making all of this up so the details of this might be wrong, but you get the idea. :)

Reply:
done